### ISSUE TITLE
How to get content from RSS 1.0

### ISSUE THREAD

**Issue by Roald87** [Roald87/motherfucking-rss-reader#9]
This test fails currently. The FeedReader doesn't parse content from RSS 1.0 it seems.

```
[<Fact>]
let ``Test get content for article text if description is empty from nature`` () =
    let actual = parseRssFromFile "data/nature.rss" |> List.head
    let expectedText =
        "Nature, Published online: 20 August 2024; doi:10.1038/s41586-024-07956-2 Author Correction: Anti-TIGIT antibody improves PD-L1 blockade through myeloid and Treg cells"
        
    Assert.Equal(expectedText, actual.Text)
```

**Comment by Roald87** [comment]
Content is parsed for atom feeds, see #8

**Comment by Roald87** [comment]
closed with #17
